bin/build-cpp-pcp-client.ps1

```diff
@@ -22,14 +22,14 @@ Write-Host "Starting cpp-pcp-client build"
 mkdir -Force release
 cd release
 
-## Build pxp_agent
+## Build cpp_pcp_client
 $cmake_args = @(
   '-G',
   "MinGW Makefiles",
   "-DBOOST_ROOT=`"$toolsDir\$boostPkg`"",
   "-DBOOST_STATIC=ON",
   "-DYAMLCPP_ROOT=`"$toolsDir\$yamlPkg`"",
```

Comment:
We decided to ignore the cpplint warning during the pxp-agent windows build development.